testdata/scripts/lines.txt

```diff
@@ -0,0 +1,62 @@
+garble -debugdir=.obf-src build
+
+env TINY_PATTERN='^\/\/line :1$'
+env DEFAULT_PATTERN='^\/\/line \w\.go:[1-9][0-9]*$'
+env DEFAULT_STACK_PATTERN='^\t\w\.go:[1-9][0-9]*(\s\+0x[0-9a-f]+)?'
+env TINY_STACK_PATTERN='^\t\?\?:[0-9][0-9]*(\s\+0x[0-9a-f]+)?$'
+
+# Default mode
+
+# Check for file name leak protection
+grep $TINY_PATTERN $WORK/.obf-src/main/main.go
+grep $TINY_PATTERN $WORK/.obf-src/main/main1.go

```

Comment:
Do you need `$WORK/foo` instead of just `foo`? because we don't ever `cd` into a different directory